C_2.scala:4: error: not found: value println
  println("fail")
  ^
one error found
